PowerPointLabs/PowerPointLabs/Views/HighlightLabSettingsDialogBox.xaml.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Windows.Forms;
+using System.Windows.Media;
+
+using PowerPointLabs.Utils;
+
+namespace PowerPointLabs.Views
+{
+    /// <summary>
+    /// Interaction logic for HighlightLabSettingsDialogBox.xaml
+    /// </summary>
+    public partial class HighlightLabSettingsDialogBox
+    {
+        public delegate void UpdateSettingsDelegate(System.Drawing.Color highlightColor, 
+                                                    System.Drawing.Color defaultColor, 
+                                                    System.Drawing.Color backgroundColor);
+        public UpdateSettingsDelegate SettingsHandler { get; set; }
+
+        public HighlightLabSettingsDialogBox()
+        {
+            InitializeComponent();
+        }
+
+        public HighlightLabSettingsDialogBox(System.Drawing.Color defaultHighlightColor, 
+                                            System.Drawing.Color defaultTextColor, 
+                                            System.Drawing.Color defaultBackgroundColor)
+            : this()
```

Comment:
You can import both colors and give the imports a name to differentiate them.
Something like this? using Drawing = System.Drawing (You can think of a better name)